selfdrive/car/hyundai/values.py

```diff
@@ -1940,6 +1942,23 @@ def match_fw_to_car_fuzzy(live_fw_versions) -> Set[str]:
       b'\xf1\x00MQhe SCC FHCUP      1.00 1.07 99110-P4000         ',
     ],
   },
+  CAR.AZERA_7TH_GEN: {
```

Comment:
This is actually the 6th gen, you can tell by the platform code.

![image](https://github.com/commaai/openpilot/assets/25857203/19a889c0-70f6-412f-8083-afa138ac9cc7)

Reply:
This is very odd.

Reply:
What is odd?